pkg/storaged/content-views.jsx

```diff
@@ -526,25 +531,28 @@ function block_content(client, block, allow_partitions) {
     var format_disk_btn = null;
     if (allow_partitions)
         format_disk_btn = (
-            <div className="pull-right">
+            <div className="pull-right" key="create-partition-table">
                 <StorageButton onClick={format_disk} excuse={block.ReadOnly ? _("Device is read-only") : null}>
                     {_("Create partition table")}
                 </StorageButton>
             </div>);
 
     return (
         <CockpitListing.Listing title={_("Content")}
-                                actions={format_disk_btn}>
+                                actions={[ format_disk_btn ]}
+                                emptyCaption={_("No entries")}>
             { block_rows(client, block) }
         </CockpitListing.Listing>
     );
-}
+};
 
-var Block = React.createClass({
-    render: function () {
-        return block_content(this.props.client, this.props.block, this.props.allow_partitions !== false);
-    }
-});
+const Block = ({client, block, allow_partitions}) => {
+    return (
+        <BlockContent client={client}
+                      block={block}
+                      allow_partitions={!allow_partitions} />
```

Comment:
`!allow_partitions` is not the same as `allow_partitions !== false`.

Reply:
oh, right, thanks